user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md

```diff
@@ -28,41 +27,63 @@ The predicted trend is displayed as a line surrounded by three shaded areas:
 
 > [!NOTE]
 >
-> - The probability percentages are based on the assumptions made at the moment the graph is generated. If the behavior of the data changes afterwards, the probability may need to be adjusted.
-> - If a recurring pattern is found, the predicted trend is based on this pattern. Otherwise, the predicted trend should be considered as the average future value.
+> - The predicted trend is generated by a machine learning model fit on the historical trend data of the metric.
+> - The probability percentages are based on the model made at the moment the graph is generated. If the behavior of the data changes afterwards, the probability may need to be adjusted.
 
 Different types of trend prediction can be displayed, each in a different style.
 
 From DataMiner 10.0.12 onwards:
 
-- High-precision prediction is displayed as a dotted line.
+- **High-precision prediction** is displayed as a dotted line.
 
-- Short-term prediction is displayed with short dashes.
+  A high-precision prediction is generated by a model fit on the real-time trend data. This type of trend prediction offers the greatest level of detail but also has the shortest forecast time horizon.
 
-- Mid-term prediction is displayed with long dashes.
+- **Short-term prediction** is displayed with short dashes.
 
-- Long-term prediction is displayed with slightly longer dashes
+  A short-time prediction (prior to DataMiner 10.0.12: mid-term prediction) is generated by a model fit on the 5-minute averaged trend data. The use of 5-minute averages usually over extended time spans means that the forecast time horizon for a short-term prediction is typically longer compared to a high-precision prediction.
+
+- **Mid-term prediction** is displayed with long dashes.
+
+  A mid-term prediction is generated by a model fit on the 1-hour averaged trend data. This type of trend prediction offers a forecast time horizon that is typically longer compared to a short-term prediction.
+
+- **Long-term prediction** is displayed with slightly longer dashes.
+
+  A long-term prediction is generated by a model fit on an aggregated set of 1-hour trend data points. This type of trend prediction extrapolates the global trend in the data to the future.
```

Comment:
```suggestion
  A long-term prediction is generated by a model fit on an aggregated set of 1-hour trend data points. This type of trend prediction extrapolates the global trend in the data to make forecasts for the future.
```

Is this different wording correct @VeerleLedoux ?

Reply:
I am not sure your wording expresses exactly what I meant.  I mainly wanted to make clear that the long-term prediction will just show the global long-term trend line in the data and will not contain the more short-term fluctuactions etc